### ISSUE TITLE
Error at start - ERROR: SyntaxError: Unexpected token '', ""... is not valid JSON

### ISSUE THREAD

**Issue by Slumpedd** [alexemanuelol/rustplusplus#308]
I had the bot working for a few weeks and made no changes. The past few days I have been trying to turn it on. I navigate to the rustplusplus directory and then do npm start run and this is the output:

PS C:\Users\brian> cd rustplusplus
PS C:\Users\brian\rustplusplus> npm start run

> rustplusplus@1.15.0 start
> ts-node . run

2024-3-26 15:23:41 ERROR: SyntaxError: Unexpected token '', ""... is not valid JSON
PS C:\Users\brian\rustplusplus>

Any suggestions?

**Comment by alexemanuelol** [comment]
Something got messed up in your instance file or credentials file. most likely your instance file. The quick solution is to remove it completely from the instance/ folder and just restart the bot. You could also run the instance_invalid branch to print out more information regarding every write to the instance file to figure out what happened.

Did your PC/Server crash and then the bot stopped working?

**Comment by Slumpedd** [comment]
Ok thanks - so I deleted the instances and now the bot is booting up but when I run npm start run this is the output:
> rustplusplus@1.15.0 start
> ts-node . run

2024-3-29 19:50:25 INFO: LOGGED IN AS: rustplusplus#9728
2024-3-29 19:50:27 INFO: Successfully registered application commands for guild: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
2024-3-29 19:50:31 INFO: FCM-listener Host will start in 5 seconds for guildId: xxxxxxxxxxxxxxxxxxxxxxx, steamId: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx.

Tried repairing to the server and am not having any luck. Is there a step I am missing? I do not have much/any coding experience..
